.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 hugo
 .idea/
+.vscode/
```

Comment:
Instead of editing this .gitignore
You can have a global gitignore which is better http://egorsmirnov.me/2015/05/04/global-gitignore-file.html